### DISCUSSION TITLE
Hobby Account - Import Repository from an Organization

### DISCUSSION THREAD

**Discussion by developh-co** [vercel/community#6092] [Feature Question] [answered]
### Summary

Hey everyone,

I would like to know if it is possible for a Hobby account to import a github repository that is hosted on an organization that this Hobby account is member of,

Note: I saw that Hobby accounts are going to get a Free team to ease the process of importing between hobby and team accounts.

With this, what's happening to this issue?

Thank you for reading, I would like to get some help,

Tiago

### Steps to Reproduce

Vercel Dashboard -> Add New -> Project

**Comment by amyegan** [comment, marked as answer]
Good question! The shift from [personal accounts to hobby teams](https://vercel.com/changelog/2024-01-account-changes) just makes things run more smoothly when upgrading or downgrading plan type. All of the [hobby plan](https://vercel.com/docs/accounts/plans/hobby#) allowances and limits still apply to hobby teams. Your hobby team will be able to connect to your personal GitHub repos but not to organization repos.

**Comment by versecafe** [comment]
@develophpt You can always fork the project to your personal account and run the fork on the hobby plan, but I do not know if this is against Vercel's policies that would be something to ask @amyegan about.
